participants/IPFS-Found.md

```diff
@@ -0,0 +1,66 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name : IPFS Found
+- Slack Handles : IPFSUnion.cn
+
+## (1) Brief Project Description
+
+Based on Filecoin network, we develop an application platform for storage and retrieval to help users store various types of files, including text, picture, audio, video, etc. IPFS-Found supports storage of multiple data sizes. We aim to build more decentralized, stable and robust data storage platform.
+
+## (2) Link to Project UI
+
+Link : https://filecoin.in
+
+## (3) What does your application/UI do?
+
+This is a completely data storage and retrieval platform，which supports the following features: 
+- Storage Client: user can easily storage the real data to the Filecoin network by "IPFS Found", including different types of data.
+- Retrieval Client: user can query the desired files by the file name or file CID.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+It will use many kinds of data from "curated" dataset.
```

Comment:
Can you be more specific here?

Reply:
We use "COVID-19 Open Research Dataset", "Chest X-Ray Images (Pneumonia)", "Huge Stock Market Dataset", "Open Images Dataset" and "Filecoin Proofs" in the Slingshot Competiton.